scene/main/canvas_item.cpp

```diff
@@ -364,7 +364,7 @@ void CanvasItem::_propagate_visibility_changed(bool p_visible) {
 	if (p_visible) {
 		update(); //todo optimize
 	} else {
-		emit_signal(SceneStringNames::get_singleton()->hide);
+		emit_signal("hidden");
```

Comment:
`SceneStringNames::hide` is now unused and should be removed - or it should be renamed `hidden` and kept here. `SceneStringNames` is used to use pre-defined StringNames instead of constructing them on the fly in performance critical code.

Reply:
Changed `SceneStringNames::hide` to `SceneStringNames::hidden`, and configured it to use it for emitting the signal.